tests/test_vnet.py

```diff
@@ -3171,6 +3171,30 @@ def test_vnet_orch_22(self, dvs, testlog):
         check_remove_state_db_routes(dvs, 'Vnet12', "100.100.1.67/32")
         #adv should be gone.
         check_remove_routes_advertisement(dvs, "100.100.1.0/24")
+
+		#Add priority route with no secondary enpoints
+        create_vnet_routes(dvs, "100.100.1.71/32", vnet_name, '19.0.0.1,19.0.0.2', ep_monitor='19.1.0.1,19.1.0.2', profile = "test_prf", primary ='19.0.0.1,19.0.0.2',monitoring='custom', adv_prefix='100.100.1.0/24')
```

Comment:
Lets have this test with both endpoint and endpoint monitor the same ips.

Reply:
sure, ill change it.